File: simuladordeinvestimento.py
capital = float(input("Valor inicial: R$ "))
taxa = float(input("Taxa de juros anual (%): "))
anos = int(input("Número de anos: "))

for i in range(1, anos + 1):
    capital += capital * (taxa / 100)
    print(f"Ano {i}: R$ {capital:.2f}")

File: simuladordeinvestimento.c
#include <stdio.h>

int main() {
    double valor_inicial, taxa;
    int anos, i;
    double valor_atual;

    printf("Valor inicial: ");
    scanf("%lf", &valor_inicial);
    printf("Taxa de juros anual (%%): ");
    scanf("%lf", &taxa);
    printf("Numero de anos: ");
    scanf("%d", &anos);

    valor_atual = valor_inicial;
    taxa = taxa / 100.0;  // converter para decimal

    printf("\n=== EVOLUCAO DO INVESTIMENTO ===\n");
    for (i = 1; i <= anos; i++) {
        valor_atual = valor_atual * (1 + taxa);
        printf("Ano %d: %.2f\n", i, valor_atual);
    }

    return 0;
}